code/campaign/src/androidTest/java/com/adobe/marketing/mobile/campaign/MonitorExtension.kt

```diff
@@ -0,0 +1,10 @@
+package com.adobe.marketing.mobile.campaign
```

Comment:
add copyright header

Reply:
Wasn't actually using this class, deleted it